bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -59,9 +59,43 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.initContainers }}
+      {{- if or .Values.initContainers .Values.auth.tls.enabled }}
       initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.auth.tls.enabled }}
+        - name: init-certs
+          image: busybox
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+          - /bin/sh
+          - -c
+          - |-
+            ID="${MY_POD_NAME#${MY_INSTANCE_NAME}-}"
```

Comment:
I'm not sure `$MY_INSTANCE_NAME` is equivalent to `$fullname := include "keycloak.fullname" .`. If you take a look at the helper's logic, there are cases when these 2 envs are different and this would make `$ID` have a wrong value. I think it would be best to leave this as `$fullname`, WDYT?

Reply:
I'll revert back to original logic to keep it simple. I was trying to remove a dependency on Helm, but if there is a potential for conflict it's not worth the refactor.